### ISSUE TITLE
Aplicações Mobile

### ISSUE THREAD

**Issue by AmiltonRN** [Samuel-Oliveira/Java_NFe#121]
É possível usar essa Lib com  Android(Java/Kotlin)

**Comment by resplandes** [comment]
Sim, eu só não indico tu acoplar isso ao android, faça uma API com essa Bliblioteca, pode ser usando springboot mesmo, e consuma ela com seu android, será bem mais viável.

**Comment by Samuel-Oliveira** [comment]
Se você conseguir fazer o Android rodar com JAva 8, mas compatibilidade com todas as dependencias.
Sim, você pode.

Mas como o Will Disse acima, não seria a melhor implementação. Um WebService com Rest é o Ideal.

**Comment by rreynoud** [comment]
Nos implementamos isso com uma API, eu acho que seja a melhor forma, ate mesmo com relação a sequencia do nro da nota fiscal e segundo porque a receita sempre esta mudando e em uma api vc modifica "uma vez e funciona para todo mundo", qualquer coisa se precisar de ajuda é so avisar.

**Comment by Samuel-Oliveira** [comment]
Duvida foi respondida? Podemos fechar?
